### DISCUSSION TITLE
[Resolved] Pls. Support -l/--stdin-line Option of Mosquitto_Pub

### DISCUSSION THREAD

**Discussion by yasurok** [termux/termux-packages#7434] [Ideas]
The mosquitto_pub command of the Mosquitto package does not seem to support [the `-l`/`--stdin-line` command option](https://mosquitto.org/man/mosquitto_pub-1.html#:~:text=8883.-,-l%2C,--stdin-line,-Send).

```
~ $ mosquitto_pub  -l
Error: '-l' mode not available, threading support has not been compiled in.
```

Sure, you can work around that relatively easily, but it's a nice feature to have. Would you support it please if you could?

Thanks!

Yasuro

**Comment by MrAdityaAlok** [comment]
#7435 should resolve this

**Reply by ghost** [reply]
Yes, resolved.
